Update verification process for helm charts and policies (#156)
kubewarden/docs: docs/security/verifying-kubewarden.md

You can inspect the cert signature yourself to see that indeed was authenticated
via GitHub OIDC, and performed in our GitHub Actions workflows.

## Policies

> Policies maintained by the Kubewarden team are also signed via Sigstore. As seen for

Reviewer: Slightly rephrasing it to make the sentence sound idiomatic.
```suggestion
Policies maintained by the Kubewarden team are also signed using the Sigstore project. Similar to
```

usual container images, one can verify them using `cosign`:
```

> vic@viccuad2 1 ~/suse/kw/kubewarden-controller (main $%>)$ COSIGN_EXPERIMENTAL=1 cosign verify ghcr.io/kubewarden/policies/verify-image-signatures:v0.2.0

Author: ```suggestion
COSIGN_EXPERIMENTAL=1 cosign verify ghcr.io/kubewarden/policies/verify-image-signatures:v0.2.0
```
